app/views/publishers/settings/index.html.slim

```diff
@@ -1,20 +1,33 @@
+.single-panel--wrapper.single-panel--wrapper--large.single-panel--wrapper--short
+  .single-panel--padded-content--short-padding
+    h5= t ".header"
+    = form_for(current_publisher, url: publisher_path(current_publisher.id), method: :patch) do |f|
```

Comment:
It's interesting to me that we have this form linking to the publishers_controller but this view is in the settings_controller. Seems like this might be a separation of concerns problem?

Reply:
To me it makes most sense since it's an update to the `publishers` table/model. `settings_controller` in retrospect should be `#edit`. Thoughts?

Reply:
I'm good with that